Source/SwiftLintFramework/Rules/StatementPositionRule.swift

```diff
@@ -28,15 +28,59 @@ public struct StatementPositionRule: Rule {
             "}↓  else {",
             "}↓\ncatch {",
             "}\n\t↓  catch {"
+        ],
+        corrections: [
+          "}\n else {\n":"} else {\n",
```

Comment:
elsewhere in the code, we use a single space after the colon in dictionaries: `"{\n": "}"`